frontend/src/Schedulr.js

```diff
@@ -15,38 +15,55 @@ class Schedulr extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      activeStep: 0,
-      steps: ['Input Courses and Preferences', 'View Schedules', 'Export to Google Calendar'],
+      activeStep: (new URLSearchParams(window.location.search).get('signedIn') === 'true') ? 1 : 0,
+      steps: ['Sign In', 'Input Courses and Preferences', 'View Schedules', 'Export to Google Calendar'],
       scheduleList: [],
       schedulesTimes: [],
+      calId: '',
+      termDates: {
+        startDate: '',
+        endDate: '',
+      },
     };
 
     this.handleNext = this.handleNext.bind(this);
     this.handleBack = this.handleBack.bind(this);
     this.setScheduleList = this.setScheduleList.bind(this);
     this.generateScheduleMeetingTimes = this.generateScheduleMeetingTimes.bind(this);
+    this.exportToGoogleCalendar = this.exportToGoogleCalendar.bind(this);
+    this.setTermDates = this.setTermDates.bind(this);
   }
 
   getStepContent() {
     const step = this.state.activeStep;
     switch (step) {
     case 0:
+      return (
+        <Button onClick={this.signIn}>Get Started</Button>
+      );
+    case 1:
       return (<InputForm
         handleNext={this.handleNext}
-        setScheduleList={this.setScheduleList}/>);
-    case 1:
+        setScheduleList={this.setScheduleList}
+        setTermDates={this.setTermDates}/>);
+    case 2:
       return (<Calendar
         scheduleList={this.state.scheduleList}
-        schedulesTimes={this.state.schedulesTimes}/>);
-    case 2:
-      return 'Export to google calendar: coming soon ;)';
+        schedulesTimes={this.state.schedulesTimes}
+        exportToGoogleCalendar={this.exportToGoogleCalendar}/>);
+    case 3:
+      return (<iframe title="google calendar" id="calendar" src={this.state.calId} width="800" height="600" frameBorder="0" scrolling="no"></iframe>);
     default:
       return 'Unknown step';
     }
   }
 
+  signIn() {
+    window.location.assign('https://course-scheduler-step-2020.uc.r.appspot.com/auth');
```

Comment:
You shouldn't hard-code the URL here. If you deploy this site at a different address, it will break. What you really want is a relative path:

```
window.location.assign('/auth');
```

Reply:
Got it, thanks for letting me know!